rundeckapp/grails-spa/packages/ui-trellis/src/app/components/activity/activityList.vue

```diff
@@ -134,9 +145,11 @@
 
     <modal
       id="bulkexecdelete"
+      ref="bulkexecdeleteresult"
```

Comment:
please avoid using ref if it's just for tests

Reply:
> please avoid using ref if it's just for tests

I have used data-tested before but it was not able to find modal. That's why ended up using ref.